<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>320</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::WheelFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color:#3a3b3a;
	color:#ffffff;
	border-color:#3a3b3a;
	border-style: solid;
	border-width:2px;
}
QLineEdit{
	background-color:#ffffff;
	color:#3a3b3a;
	border-color:#3a3b3a;
	border-style:solid;;
	border-width:2px;
	height:25px;
	font:bold
}
QLineEdit::focus{
	border-style:solid;
	border-width:2px;
	border-color:#54b0cb;
}
QLabel{
	background-color:#3a3b3a;
	border:0px solid #3a3b3a;
	padding:5px
}
QPushButton{
	background-color:#54b0cb;
	color:#ffffff;
	font:bold;
	height:30px;
	border-color:#54b0cb;
}
QPushButton::hover{
	background-color:#448fa6;
	border-color:#448fa6;
}
QPushButton::focus{
	background-color:#448fa6;
}
QPushButton::pressed{
	background-color:#3a3b3a;
	color:#ffffff;
	font:bold;
	border-style:solid;
	border-color:#ffffff;
	border-width:2px;
}
QComboBox{
	background-color:#ffffff;
	border-color:#3a3b3a;
	border-style:solid;;
	border-width:2px;
	color:#3a3b3a;
	width:50px;
	height:30px;
	font:bold;
	padding-left:5px
}
QComboBox::focus{
	border-style:solid;
	border-width:2px;
	border-color:#54b0cb;
}
QComboBox QAbstractItemView {
	background-color:#ffffff;
	color:#3a3b3a;
    border: 2px solid #ffffff;
	padding-left:2px;
	padding-right:2px;
	padding-top:2px;
	padding-bottom:2px
}
QComboBox::drop-down {
   width: 24px;
   border: 2px;
}
QComboBox::down-arrow {
    image: url(&quot;/home/krsnvss/beetcount/operators_gui/res/if_chevron-left.png&quot;);
    width: 16px;
    height: 16px;
}
QComboBox::down-arrow::hover {
    image: url(&quot;/home/krsnvss/beetcount/operators_gui/res/if_chevron-down.png&quot;);
    width: 16px;
    height: 16px;
}
QTableView{
	background-color:#ffffff;
	alternate-background-color:#e2e2e2;
	color:#3a3b3a;
	selection-color:#ffffff;
	selection-background-color:#54b0cb;
	border:2px solid #ffffff;
}
QHeaderView::section{
    background-color:#448fa6;
	color:#ffffff;
	font:bold;
	height:40px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="headerEdit">
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	background-color:#54b0cb;
	color:#ffffff;
	font:bold;
	border-color:#54b0cb;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="docEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="placeholderText">
      <string>Номер документа</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="driverEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Водитель</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="infoLbl">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Автомобиль: &lt;span style=&quot; font-weight:600;&quot;&gt;А124АА48&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Прицеп: &lt;span style=&quot; font-weight:600;&quot;&gt;АА1245&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Поставщик: &lt;span style=&quot; font-weight:600;&quot;&gt;ООО &amp;quot;Прима&amp;quot;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="loadEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Пункт погрузки</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="unloadsList"/>
   </item>
   <item>
    <widget class="QPushButton" name="saveBtn">
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color:#54b0cb;
	color:#ffffff;
	font:bold;
	height:30px;
	border-color:#54b0cb;
}
QPushButton::hover{
	background-color:#448fa6;
	border-color:#448fa6;
}
QPushButton::focus{
	background-color:#448fa6;
}
QPushButton::pressed{
	background-color:#3a3b3a;
	color:#ffffff;
	font:bold;
	border-style:solid;
	border-color:#ffffff;
	border-width:2px;
}</string>
     </property>
     <property name="text">
      <string>Сохранить изменения</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="changeDriver">
   <property name="icon">
    <iconset>
     <normaloff>../res/if_edit.png</normaloff>../res/if_edit.png</iconset>
   </property>
   <property name="text">
    <string>Изменить водителя</string>
   </property>
   <property name="toolTip">
    <string>Изменить водителя</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+D</string>
   </property>
  </action>
  <action name="changeLoad">
   <property name="icon">
    <iconset>
     <normaloff>../res/if_edit.png</normaloff>../res/if_edit.png</iconset>
   </property>
   <property name="text">
    <string>Изменить пункт погрузки</string>
   </property>
   <property name="toolTip">
    <string>Изменить пункт погрузки</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+L</string>
   </property>
  </action>
  <action name="closeForm">
   <property name="icon">
    <iconset>
     <normaloff>../res/if_x_white.png</normaloff>../res/if_x_white.png</iconset>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
   <property name="toolTip">
    <string>Закрыть форму редактирования рейса</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="saveChanges">
   <property name="icon">
    <iconset>
     <normaloff>../res/if_save_white.png</normaloff>../res/if_save_white.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="toolTip">
    <string>Сохранить изменения</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
